src/misc/troubleshoot.md

```diff
@@ -40,24 +40,18 @@ Sometimes it is useful to share your project with our support team so that they
 
 To share your <MainPlatformName /> project with the support:
 1. Log in to <AppDomainNameLink />
-2. In the **Project** tab in the left panel, find the project that needs to be shared with the support
-   ![List of projects in Mergin Maps](./mergin-find-project.jpg "List of projects in Mergin Maps")
+2. In the **Project** tab in the left panel, find the project that needs to be shared with the support and click on it.
```

Comment:
Edit text:
In the **Project** tab on the left panel, find the project that needs to be shared with the support team and click on it.